system/include/emscripten/fetch.h

```diff
@@ -65,6 +64,16 @@ extern "C" {
 // fetch to test or wait for its completion.
 #define EMSCRIPTEN_FETCH_WAITABLE 128
 
+// If specified, the request will be considered a high priority fetch request relative to other requests of the same type.
+// EMSCRIPTEN_FETCH_HIGH_PRIORITY and EMSCRIPTEN_FETCH_LOW_PRIORITY are mutually exclusive.
+// If neither is provided, the request priority will be "default"
+#define EMSCRIPTEN_FETCH_HIGH_PRIORITY 256
+
+// If specified, the request will be considered a low priority fetch request relative to other requests of the same type.
+// EMSCRIPTEN_FETCH_HIGH_PRIORITY and EMSCRIPTEN_FETCH_LOW_PRIORITY are mutually exclusive.
+// If neither is provided, the request priority will be "default"
+#define EMSCRIPTEN_FETCH_LOW_PRIORITY 512
```

Comment:
To keep this change simpler I wonder if we could to the transition and then add these extra settings as a followup?   

That would make this change effect very few files, and mean that we could discuss this as essentially a non-functional change?